### DISCUSSION TITLE
How do I change the editor?

### DISCUSSION THREAD

**Discussion by DigDogData** [nvim-neorg/neorg#647] [Q&A]
I have only been using Neorg since yesterday, lot to learn - awesome plugin! I have one quesion: pressing the 'Enter' key on a file path link in my `index.norg` opens the file in the Xed text editor, which I occasionally use, even though Neovim is set as default (`echo $EDITOR` shows `nvim`). I'd like to open the file in Neovim. Is there a way to do this?

Thanks!

**Comment by miraculusik** [comment]
Can you try this in `.zshrc` -> `export EDITOR="$(which nvim)"`
